common/components/AddressFieldFactory/AddressFieldFactory.tsx

```diff
@@ -15,16 +16,28 @@ interface OwnProps {
   withProps(props: CallbackProps): React.ReactElement<any> | null;
 }
 
+interface State {
+  isFocused: boolean;
+}
+
 export interface CallbackProps {
   isValid: boolean;
   readOnly: boolean;
   currentTo: ICurrentTo;
   onChange(ev: React.FormEvent<HTMLInputElement>): void;
+  onFocus(ev: React.FormEvent<HTMLInputElement>): void;
```

Comment:
The onFocus for the address input should be updated so it isn't `invalid` when searching for an address in your address book. 
<img width="912" alt="screen shot 2018-05-01 at 3 33 27 pm" src="https://user-images.githubusercontent.com/13836920/39489957-305a44ae-4d55-11e8-8174-131852a7f970.png">


This occurs after:
- Begin searching for a valid address
- leave the input (`onBlur()`)
- focus the input (`onFocus()`)

Reply:
Also, if you start typing an address label that doesn't exist, it should display an error, something like 'That address doesn't exist in your address book'.

Reply:
Agree on the bottom, but I was told that an I out should show invalid if it’s invalid. Even if you’re searching for a label, at that moment it’s not actually valid.

Clarification, @dternyak ?

Reply:
@ConnorBryan The input that I mentioned that should have invalid styling when invalid is the address book address input, not the To Address input.

Reply:
@dternyak

Okay! I’ll get this fixed up when I get home from the hospital later.